packages/cli/commands/init.js

```diff
@@ -105,10 +104,14 @@ exports.describe = i18n(`${i18nKey}.describe`, {
 exports.handler = async options => {
   const {
     auth: authType = PERSONAL_ACCESS_KEY_AUTH_METHOD.value,
-    c,
     account: optionalAccount,
   } = options;
-  const configPath = (c && path.join(getCwd(), c)) || getConfigPath();
+  const { useNewConfig } = await promptUser({
```

Comment:
This prompt may be confusing to users who are new to the CLI and have no idea what new or old versions of the config are.

One way we could slowly and safely roll this out could be to initially support a  hidden `--useHomeConfig` flag (or something along those lines) that routes to the new config logic when true. That way we won't need to add a prompt.

Then, later down the road and after tome validation testing, we could invert that flag so that the default behavior would be to use the new home config. We could then support some sort of `--useLegacyConfig` so devs who aren't ready to use the new version of config can still have access to the old version.

Reply:
The major pro of a pattern like this would be that we can roll it out without it being a major breaking change.

Reply:
Will take the suggestion, thanks! Just added the prompt because I knew we needed some kind of user input, but didn't think of a flag 🤦‍♀️

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1123/commits/2296e84cb12ad2d78c96ce5e924f861afdfa5200